solvers/DPStokes/mobility.h

```diff
@@ -51,23 +52,58 @@ class DPStokes: public libmobility::Mobility{
     this->lanczosTolerance = ipar.tolerance;
     this->dppar.mode = this->wallmode;
     this->dppar.hydrodynamicRadius = ipar.hydrodynamicRadius[0];
-    this->dppar.w = 6;
-    this->dppar.beta = 1.714*this->dppar.w;
-    real h = this->dppar.hydrodynamicRadius/1.554;
+    this->dppar.w = 4;
+    this->dppar.beta = 1.785*this->dppar.w;
+    real h = this->dppar.hydrodynamicRadius/1.205;
     this->dppar.alpha = this->dppar.w/2.0;
     this->dppar.tolerance = 1e-6;
-    this->dppar.nx = int(this->dppar.Lx/h + 0.5);
-    this->dppar.ny = int(this->dppar.Ly/h + 0.5);
-    // Add a buffer of w*h/2 when there is an open boundary
+
+    real N_real = this->dppar.Lx/h; // actual N given L and h
+    int N_up = ceil(N_real);
+    int N_down = floor(N_real);
+    int N;
+    // either N_up or N_down will be a multiple of 2. pick the even one for a more FFT friendly grid.
+    if(N_up % 2 == 0){
+      N = N_up;
+    }else{
+      N = N_down;
+    }
+
+    this->dppar.nx = N;
+    this->dppar.ny = N;
```

Comment:
You can rewrite as:
```c++
    int N = floor(this->dppar.Lx / h);
    N+= N % 2;
    this->dppar.nx = this->dppar.ny = N;
```
BTW, is the fact that Ly is ignored on purpose? Are we supporting only Lx=Ly or something like that?

Reply:
Good catch- I originally wrote this supporting only Lx=Ly as the parameter selection code from DPStokes did that. I think @brennansprinkle mentioned it was likely because having Nx=Ny is the best for efficiency reasons related to FFTs, but I might be remembering incorrectly. 

This routine could be modified to support Lx != Ly. We could try this if it isn't too complicated:

1. For a general Lx and Ly, try and compute as before leaving the box size unchanged. This will produce an $h_x$ and $h_y$ that aren't going to be the same, in general. 
2. If $h_x = h_y$, proceed as normal. If not, throw an error that says you must either set Lx = Ly or allow us to change the box size. If Lx = Ly, then we're guaranteed that $h_x = h_y$. If we're allowed to change the box size, we let $h = \min(h_x, h_y)$ and then modify one of Lx or Ly to make Nx= Lx/h and Ny = Ly/h integers. This would give us a uniform grid spacing but with a different number of nodes in each direction.

Should I implement the above parameter selection or simply require that Lx = Ly?